stories/object-status/object-status.stories.js

```diff
@@ -349,64 +279,89 @@ export const inverted = () => `
-    <span class="fd-object-status__text">Indication1</span>
-</span>
-<span class="fd-object-status fd-object-status--inverted fd-object-status--link fd-object-status--indication-2">
-    <span class="fd-object-status__text">Indication2</span>
-</span>
-<span class="fd-object-status fd-object-status--inverted fd-object-status--link fd-object-status--indication-3">
-    <span class="fd-object-status__text">Indication3</span>
-</span>
-<span class="fd-object-status fd-object-status--inverted fd-object-status--link fd-object-status--indication-4">
-    <span class="fd-object-status__text">Indication4</span>
-</span>
-<span class="fd-object-status fd-object-status--inverted fd-object-status--link fd-object-status--indication-5">
-    <span class="fd-object-status__text">Indication5</span>
-</span>
-<span class="fd-object-status fd-object-status--inverted fd-object-status--link fd-object-status--indication-6">
-    <span class="fd-object-status__text">Indication6</span>
-</span>
-<span class="fd-object-status fd-object-status--inverted fd-object-status--link fd-object-status--indication-7">
-    <span class="fd-object-status__text">Indication7</span>
-</span>
-<span class="fd-object-status fd-object-status--inverted fd-object-status--link fd-object-status--indication-8">
-    <span class="fd-object-status__text">Indication8</span>
-</span>
+<div class="fddocs-container">
+    <span class="fd-object-status fd-object-status--inverted fd-object-status--link fd-object-status--indication-1">
+        <span class="fd-object-status__text">Indication1</span>
+    </span>
+    <span class="fd-object-status fd-object-status--inverted fd-object-status--link fd-object-status--indication-2">
+        <span class="fd-object-status__text">Indication2</span>
+    </span>
+    <span class="fd-object-status fd-object-status--inverted fd-object-status--link fd-object-status--indication-3">
+        <span class="fd-object-status__text">Indication3</span>
+    </span>
+    <span class="fd-object-status fd-object-status--inverted fd-object-status--link fd-object-status--indication-4">
+        <span class="fd-object-status__text">Indication4</span>
+    </span>
+    <span class="fd-object-status fd-object-status--inverted fd-object-status--link fd-object-status--indication-5">
+        <span class="fd-object-status__text">Indication5</span>
+    </span>
+    <span class="fd-object-status fd-object-status--inverted fd-object-status--link fd-object-status--indication-6">
+        <span class="fd-object-status__text">Indication6</span>
+    </span>
+    <span class="fd-object-status fd-object-status--inverted fd-object-status--link fd-object-status--indication-7">
+        <span class="fd-object-status__text">Indication7</span>
+    </span>
+    <span class="fd-object-status fd-object-status--inverted fd-object-status--link fd-object-status--indication-8">
+        <span class="fd-object-status__text">Indication8</span>
+    </span>
```

Comment:
I removed some of the `rtl` examples in here - once they were using layouts instead of `br` and override styling, rtl doesn't really mean anything with this component.